FlowCrypt/Functionality/Services/EKMVcHelper.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  EKMVcHelper.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 4/28/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol EKMVcHelperType {
+    func refreshKeysFromEKMIfNeeded(in viewController: UIViewController)
+}
+
+final class EKMVcHelper: EKMVcHelperType {
+
+    let appContext: AppContextWithUser
+    let keyMethods: KeyMethodsType
+
+    init(appContext: AppContextWithUser) {
+        self.appContext = appContext
+        self.keyMethods = KeyMethods()
+    }
+
+    func refreshKeysFromEKMIfNeeded(in viewController: UIViewController) {
+        Task {
+            do {
+                // Sleep for 3 seconds when mock testing
+                // (This is to prevent refresh key UI test failure in semaphoreCI)
+                if Bundle.isDebugBundleWithArgument("--mock-fes-api") {
+                    try await Task.sleep(nanoseconds: 3 * 1000 * 1_000_000)
+                }
```

Comment:
@sosnovsky, @tomholub So Tests should pass now.
@tomholub FYI only changes I added after your approval is this logic (https://github.com/FlowCrypt/flowcrypt-ios/commit/5b1150e308137d8bc435a2212941bf41156af62c).
Let me know what you think.

```
if Bundle.isDebugBundleWithArgument("--mock-fes-api") {
  try await Task.sleep(nanoseconds: 3 * 1000 * 1_000_000)
}
```

Reply:
Quite horrible 👀 but I'm ok to merge it so we can progress.

There must be a more reasonable way to fix the issue. It could be a change to test code or production code (maybe the flow would have to be changed to be easier to test, idk).